src/jsdoc.ts

```diff
@@ -139,16 +139,27 @@ export function parse(comment: string): ParsedJSDocComment|null {
   // TODO(evanm): this is a pile of hacky regexes for now, because we
   // would rather use the better TypeScript implementation of JSDoc
   // parsing.  https://github.com/Microsoft/TypeScript/issues/7393
-  let match = comment.match(/^\/\*\*([\s\S]*?)\*\/$/);
+  const match = comment.match(/^\/\*\*([\s\S]*?)\*\/$/);
   if (!match) return null;
-  comment = match[1].trim();
+  return parseContents(match[1].trim());
+}
+
+/**
+ * parseContents parses JSDoc out of a comment text.
+ * Returns null if comment is not JSDoc.
+ *
+ * @param commentText a comment's text content, i.e. the comment w/o /* and * /.
+ */
+export function parseContents(commentText: string): {tags: Tag[], warnings?: string[]}|null {
+  // Make sure we have proper line endings before parsing on Windows.
+  commentText = normalizeLineEndings(commentText);
```

Comment:
I notice this Windows code is new (?), not sure if you meant to include it.

Reply:
I think Github's UI is fooling you, the normalizeLineEndings call has been here before.